README.md

```diff
@@ -83,18 +84,18 @@ You can also build your own by doing the following:
 ```
 git clone http://github.com/Automattic/elasticsearch-statsd-plugin.git
 cd elasticsearch-statsd-plugin
-mvn package
+mvn package -Djava.security.policy=src/test/resources/plugin-security-test.policy
 ```
 
 Once we have the artifact, install it with the following command:
 
 ```
-./bin/elasticsearch-plugin install file:///Users/anandnalya/github/automattic/elasticsearch-statsd-plugin/target/releases/elasticsearch-statsd-5.6.5.0.zip
+bin/elasticsearch-plugin install file:///absolute/path/to/current/dir/target/releases/elasticsearch-statsd-6.1.1.0.zip
 ```
 
-## Installation Elasticsearch 2.x
+## Installation Elasticsearch 2.x-5.x
 
-The plugin artifacts are published to Maven Central. To install a prepackaged plugin for ES 2.x use the following command:
+The plugin artifacts are published to Maven Central. To install a prepackaged plugin use the following command:
 
 ```
 bin/plugin install com.automattic/elasticsearch-statsd/2.4.4.0
```

Comment:
`bin/plugin install` does not work with 5.x. Maybe instead of renaming 5.x section to 6.x, just create a new section for 6.x and keep instructions for 5.x and 2.x as it is.